applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx

```diff
@@ -70,18 +70,11 @@ const VesselListItem: React.FC<ListItemProps> = (props): React.ReactElement => {
               <label>{t('vessel.transmission_plural', 'transmissions')}</label>
               {vessel.firstTransmissionDate || vessel.lastTransmissionDate ? (
                 <Fragment>
-                  {t('common.from', 'from')}{' '}
-                  {vessel.firstTransmissionDate ? (
-                    <I18nDate date={vessel.firstTransmissionDate} />
-                  ) : (
-                    DEFAULT_EMPTY_VALUE
-                  )}{' '}
-                  {t('common.to', 'to')}{' '}
-                  {vessel.lastTransmissionDate ? (
-                    <I18nDate date={vessel.lastTransmissionDate} />
-                  ) : (
-                    DEFAULT_EMPTY_VALUE
-                  )}
+                  {t('vessel.transmissionRange', 'transmissionRange', {
```

Comment:
For the default argument I would use a meaningful phrase
```suggestion
                  {t('vessel.transmissionRange', '{{transmissions}} AIS transmissions from {{start}} to {{end}}', {
```